packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -301,17 +291,10 @@ class AppBar extends StatefulWidget {
   /// bar is scrolled.
   final double bottomOpacity;
 
-  final double _bottomHeight;
-
-  /// The height of the toolbar and the [bottom] widget.
-  ///
-  /// The parent widget should constrain the [AppBar] to a height between this
-  /// and whatever maximum size it wants the [AppBar] to have.
-  ///
-  /// If [primary] is true, the parent should increase this height by the height
-  /// of the top padding specified by the [MediaQuery] in scope for the
-  /// [AppBar].
-  double get minExtent => kToolbarHeight + _bottomHeight;
+  /// A size whose height is the sum of [kToolbarHeight] and the [bottom] widget's
+  /// preferred height.
```

Comment:
maybe explicitly point to `PreferredSizeWidget.preferredSize`, saying that this implements that interface?